lib/translator/translation.cc

```diff
@@ -99,25 +99,25 @@ BeginResponse Translation::Begin(Span<const uint8_t> scsi_cmd,
     case scsi::OpCode::kRead6:
       pipeline_status_ =
           Read6ToNvme(scsi_cmd_no_op, nvme_wrappers_[0], allocations_[0], nsid,
-                      kPageSize, kLbaSize, response.alloc_len);
+                      kPageSize, kLbaSize, response.alloc_len, buffer_out);
```

Comment:
a little confusing that the parameter passed in here is called `buffer_out` but the parameter for the read_functions is called `buffer_in`. It might be helpful to add a comment somewhere to clarify. Or, if there's already a comment somewhere, then you can ignore this :)